doc/ch-image.rst

```diff
@@ -927,6 +928,156 @@ trading off the time needed to retrieve an image from cache.
    Glob characters must be quoted or otherwise protected from the shell, which
    also desires to interpret them and will do so incorrectly.
 
+:code:`export`
+===============
+
+::
+
+   $ ch-image [...] export [-o FMT] [...] IMAGE_REF PATH
+
+Description
+-----------
+
+Copy image :code:`IMAGE_REF` to :code:`PATH` and convert its format. Replace
+:code:`PATH` if it already exists, unless :code:`--no-clobber` is specified. It
+is an error if :code:`IMAGE_REF` and :code:`PATH` have the same format; use the
+format's own tools for that case.
+
+:code:`ch-run` can run container images that are plain directories or
+(optionally) SquashFS archives. However, images can take on a variety of other
+formats as well. The main purpose of this tool is to make images in those
+other formats available to :code:`ch-run`.
+
+For best performance, :code:`export` should be invoked only once,
+producing the final format actually needed.
+
+  :code:`IMAGE_REF`
+    Descriptor for the input image. For image builders, this is an image
+    reference; otherwise, it's a filesystem path.
+
+  :code:`PATH`
+    Descriptor for the output image.
+
+  :code:`-h`, :code:`--help`
+    Print help and exit.
+
+  :code:`-n`, :code:`--dry-run`
+    Don't read the input or write the output. Useful for testing format
+    inference.
```

Comment:
We should discuss whether this subcommand should infer like `ch-convert` does offline with @reidpr and friends.